homeassistant/components/broadlink/time.py

```diff
@@ -0,0 +1,70 @@
+"""Support for Broadlink device time."""
+
+from __future__ import annotations
+
+from datetime import time
+from typing import Any
+
+from homeassistant.components.time import TimeEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ServiceValidationError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.util import dt as dt_util
+
+from . import BroadlinkDevice
+from .const import DOMAIN
+from .entity import BroadlinkEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the Broadlink time."""
+    device = hass.data[DOMAIN].devices[config_entry.entry_id]
+    async_add_entities([BroadlinkTime(device)])
+
+
+class BroadlinkTime(BroadlinkEntity, TimeEntity):
+    """Representation of a Broadlink device time."""
+
+    _attr_has_entity_name = True
+    _attr_native_value: time | None = None
+
+    def __init__(self, device: BroadlinkDevice) -> None:
+        """Initialize the sensor."""
+        super().__init__(device)
+
+        self._attr_unique_id = f"{device.unique_id}-device-time"
```

Comment:
```suggestion
        self._attr_unique_id = f"{device.unique_id}-device_time"
```
_if_ you ever need to migrate you can just do `.split("-")`

Reply:
Done